client/src/components/FeedbackForm/FeedbackButton.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from 'react';
+import { Button } from 'reactstrap';
+import { useDispatch, useSelector } from 'react-redux';
+import FeedbackModal from './FeedbackModal';
+import './FeedbackButton.module.scss';
+
+const FeedbackButton = () => {
+  const [openModal, setOpenModal] = React.useState(false);
+  const creatingSuccess = useSelector(
+    state => state.ticketCreate.creatingSuccess
+  );
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    let mounted = true;
+    if (creatingSuccess && mounted) {
+      const toast = {
+        pk: 'feedback-toast',
+        event_type: 'generic',
+        message: 'Feedback submitted',
+        extra: {}
+      };
+      dispatch({
+        type: 'ADD_TOAST',
+        payload: toast
+      });
+      setOpenModal(false);
+    }
+    return () => (mounted = false);
+  }, [creatingSuccess]);
```

Comment:
This `useEffect` block runs for tickets created by both this Feedback Form and the Tickets Create modals. Either we should think about separating these events such that only this form creates a Toast, or we should plan to create toasts for all ticket submissions and adjust the toast message accordingly. @tacc-wbomar thoughts?

Reply:
@rstijerina, I'm setting a condition to create a Toast only for FeedBack modal.